oauth-common/src/main/java/io/strimzi/kafka/oauth/services/OAuthMetrics.java

```diff
@@ -90,26 +111,48 @@ private Metrics initKafkaMetrics() {
 
     private List<MetricsReporter> initReporters() {
         AbstractConfig kafkaConfig = initKafkaConfig();
-        List<MetricsReporter> reporters = kafkaConfig.getConfiguredInstances(CommonClientConfigs.METRIC_REPORTER_CLASSES_CONFIG,
-                MetricsReporter.class);
-
-        JmxReporter reporter = new JmxReporter();
-        reporter.configure(configMap);
-
-        reporters.add(reporter);
+        List<MetricsReporter> reporters = new ArrayList<>(3);
```

Comment:
Why set the starting size to 3?

Reply:
The idea is that 90% of the time people will configure at most 3 reporters. That way there will be no need to reallocate the underlying array to accommodate more. But then this is not a performance critical code and if you think it distracts from reading the code we can remove it. In all practical sense it has zero impact on performance.

Reply:
If you think this is meaningful optimization, I'd add a comment justifying why 3 was picked. When I saw it, it looked like a random number to me.

Reply:
In fact the allocated object is never added to. A remnant of refactoring ... I'll remove it.